content/en/security_platform/application_security/getting_started/java.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Java Applications
+kind: documentation
+code_lang: java
+type: multi-code-lang
+code_lang_weight: 0
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-java'
+      tag: 'GitHub'
+      text: 'Java Datadog Library source code'
+---
+
+{{% appsec-getstarted %}}
+
+## Get started
+
+1. **Download the [Datadog Java library][1]**, at least version 0.94.0:
+   ```
```

Comment:
@arnaudbreton That link [1] is pointing to exactly that URL, so they do have easy access to the latest release. We include the 0.94.0 because for at least the first few months the most common question people have is "is what I already have good enough?" and this lets them know that right away. The way the sentences is phrased ("at least") means we don't have to update it each time there's a new release. The sentence still works after 0.94.1 arrives.

Reply:
I see. Makes sense then